migrate_pim/upgrade/upgrade_from_40_to_50.rst

```diff
@@ -29,6 +29,52 @@ You have to make sure your system components are updated to the version required
 
     So there's no need to export and reimport data for this system.
 
+
+Updated System dependencies
+---------------------------
+
+Community Edition
+^^^^^^^^^^^^^^^^^
+TODO?
+
+Enterprise Edition
+^^^^^^^^^^^^^^^^^^
+The following libraries must be added to your operating system:
+
+.. code:: bash
+
+    $ apt-get install aspell-it aspell-sv aspell-da aspell-nl aspell-no aspell-pt-br
+
+Updated crontab definition
```

Comment:
Shouldn't be at the end before "Migrating your custom code"?

Reply:
you're right, the crontab definition must be done at the very end, when the code is up-to-date and the migrations done.